pyproject.toml

```diff
@@ -60,6 +60,7 @@ fsspec = ">=2022.4.0"
 packaging = ">=21.1"
 pluggy = ">=1.3.0"
 win-precise-time = {version = ">=1.4.2", markers="os_name == 'nt'"}
+sqlglot = ">=26.9.0"
```

Comment:
I took a look when dialects were added etc. and we are pretty safe with using 23.0.0 as initial version. it is a year old. why we want to put as old version as possible: people may be using other dependencies that have an upper version limit or are bound to particular version. so we want to put stuff that is as old as possible. year old version looks reasonable.